qms-tests/sanity/tests/documents/REQ-13.spec.ts

```diff
@@ -0,0 +1,199 @@
+import { test } from '@playwright/test'
+import {
+  attachScreenshot,
+  DocumentURI,
+  generateId,
+  getSecondPage,
+  HomepageURI,
+  PlatformSetting,
+  PlatformURI
+} from '../utils'
+import { allure } from 'allure-playwright'
+import { DocumentsPage } from '../model/documents/documents-page'
+import { DocumentDetails, DocumentRights, DocumentStatus, NewDocument } from '../model/types'
+import { DocumentContentPage } from '../model/documents/document-content-page'
+import { prepareDocumentStep } from '../documents/common-documents-steps'
+
+import { DocumentHistoryPage } from '../model/documents/document-history-page'
+
+test.use({
+  storageState: PlatformSetting
+})
+
+test.describe('ISO 13485, 4.2.4 Control of documents, ensure that the current revision status of and changes to documents are identified', () => {
+  test.beforeEach(async ({ page }) => {
+    await (await page.goto(`${PlatformURI}/${HomepageURI}`))?.finished()
+  })
+
+  test('TESTS-325. Create a Several documents with Minor & Major versions', async ({ page, browser }) => {
+    await allure.description(
+      'Requirement\nUsers need to make a resolve all comments and done documents for the Effective status'
+    )
+    await allure.tms('TESTS-325', 'https://front.hc.engineering/workbench/platform/tracker/TESTS-141')
+    const userSecondPage = await getSecondPage(browser)
+    const completeDocument: NewDocument = {
+      template: 'HR (HR)',
+      title: `Complete document-${generateId()}`,
+      description: `Complete document description-${generateId()}`
+    }
+    const reviewer = 'Dirak Kainin'
+    const documentDetails: DocumentDetails = {
+      type: 'HR',
+      category: 'Human Resources',
+      version: 'v0.1',
+      status: DocumentStatus.DRAFT,
+      owner: 'Appleseed John',
+      author: 'Appleseed John'
+    }
+    await prepareDocumentStep(page, completeDocument)
```

Comment:
Currently, there's an issue with concurrency handing when creating multiple documents at the same time - the system may try to give them the same ID and one of the creations may fail.
So for now I had to move all tests where documents are created to the same spec file because all tests from the same spec will run sequentially on a single worker and not in parallel.

Reply:
hmmm didn't notice that while I was making this. So suggestion is to move everything to the same spec. That might be a big spec at the end :D

Reply:
I switched fullyParallel and set workers to 1, so we shouldn't have this issue